cli/src/tests/language_test.rs

```diff
@@ -95,3 +95,67 @@ fn test_symbol_metadata_checks() {
         }
     }
 }
+
+#[test]
+fn test_supertype_data() {
```

Comment:
reminder (mostly for me): when you've updated the definition of subtypes_for_supertype to return the array of symbol ids, this test should be adjusted to map the symbol ids to their names and then compare that list